app/pages/archive.vue

```diff
@@ -97,7 +97,6 @@ export default {
           .filter(item => item.play_date < this.getToday)
```

Comment:
Implemented on Arcsi's side, we should remove in a future commit

Reply:
https://github.com/lahmacunradio/frontend/issues/255 (frontend) and https://github.com/lahmacunradio/arcsi/issues/145 (arcsi).